
Set SIGXFSZ to be ignored, so a write() beyond 2gb will fail with
E2BIG rather than killing the process

--- ./server/mpm/prefork/prefork.c.xfsz	Wed Jul 17 22:39:55 2002
+++ ./server/mpm/prefork/prefork.c	Mon Aug 26 15:40:24 2002
@@ -461,7 +461,7 @@
 	ap_log_error(APLOG_MARK, APLOG_WARNING, errno, ap_server_conf, "sigaction(SIGXCPU)");
 #endif
 #ifdef SIGXFSZ
-    sa.sa_handler = SIG_DFL;
+    sa.sa_handler = SIG_IGN;
     if (sigaction(SIGXFSZ, &sa, NULL) < 0)
 	ap_log_error(APLOG_MARK, APLOG_WARNING, errno, ap_server_conf, "sigaction(SIGXFSZ)");
 #endif
